cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/NamespaceHttpHandlerTest.java

```diff
@@ -277,6 +279,12 @@ public void testDeleteAll() throws Exception {
     Assert.assertTrue(streamAdmin.exists(myStream));
     assertResponseCode(200, deleteNamespace(OTHER_NAME));
     Assert.assertFalse(streamAdmin.exists(myStream));
+
+    assertResponseCode(200, createNamespace(OTHER_NAME));
```

Comment:
why create namespace after deleting ? Maybe we can test this before deleting the namespace, so we don;t have to create -> delete -> create -> delete?